README.md

```diff
@@ -68,7 +68,18 @@ exporter.start()
 ```javascript
 app.post('/pdfexport', function(req,res){
 	// derive job arguments from request here
-	exporter.createJob(source, target, options).then( job => {
+	// 
+	const jobOptions = {
+	  /**
+	    r.results[] will contain the following based on inMemory
```

Comment:
You may have looked at it before my last commit. Standard enforces two spaces so there is really no way for this build to succeed with tabs.